go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kubernetes-csi/external-health-monitor
 
-go 1.16
+go 1.18
```

Comment:
Changing this line does not control which version of Go it is built with.

What is the reason for changing it?